pkg/minikube/registry/drvs/docker/docker.go

```diff
@@ -37,6 +38,7 @@ import (
 )
 
 var docURL = "https://minikube.sigs.k8s.io/docs/drivers/docker/"
+var minDockerVersion = []int{18, 7, 0}
```

Comment:
lets to our documentation to let them know the min version supported is 18.7.0

https://minikube.sigs.k8s.io/docs/drivers/docker/

Reply:
Add ' 18.07 or higher' statement into `site/content/en/docs/drivers/includes/docker_usage.inc` file